### PR TITLE
Fix wayland

### PR DESCRIPTION
**Pull request by supechicken**
Tested on `x86_64`

### CODE REVIEW

File: packages/wayland.rb [hunk 1]
```diff
@@ -29,10 +29,11 @@ def self.build
...
     
-    system "meson #{CREW_MESON_FNO_LTO_OPTIONS} \
-
-    -Ddocumentation=false \
-    builddir"
+    system <<~CONFIGURE
+      meson #{CREW_MESON_FNO_LTO_OPTIONS} \
+      -Ddocumentation=false \
+      builddir
+    CONFIGURE
```
Comment (uberhacker) [inline packages/wayland.rb:36]: What exactly is fixed here?  This CONFIGURE string is not referenced anywhere else in the package.

Comment (supechicken) [reply packages/wayland.rb:36]: I removed the empty line in `line 33`

Comment (satmandu) [reply packages/wayland.rb:36]: Why use `<<~CONFIGURE` instead of just using double quotes?

Comment (uberhacker) [reply packages/wayland.rb:36]: I guess one advantage would be you don't have to escape double quotes, if needed.

**Comment by uberhacker** [comment]
Fixed in #5660.

**Pull request closed without merging**